app/controllers/admin/page_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Admin
   class PageController < BaseAdminController
+    before_action :ensure_service_team
```

Comment:
Do we have anything which tests this?

Reply:
We have the request spec that checks that a payroll operator has access and a service operator does not:

https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/1007-cantium-can-download-the-payroll-file-themselves/spec/requests/admin_payroll_run_downloads_spec.rb

Reply:
I don't think that's testing _this_ filter though - to test this one I think you'd want a spec which checks that a payroll operator is unable to access the rest of the admin interface.

Reply:
I've added a commit to the end that I am not 100% sure about, but I think it's a sounds approach, appreciate your thoughts.

There was a lot of specs already around the `support_agent` role so I added the payroll operator there. I also added a specific test for the controller using `ensure_service_team`.

Reply:
It looks good to me; I think the loop sets a pattern for people adding new request specs to follow. And only one representative example of a feature spec to avoid slowing the test suite down.